JavaScript/Script.js

```diff
@@ -1,56 +1,61 @@
 var initialMap;
 var coords = [0, 0];
 var totalWhales;
-
-function initMap(species) {
+// parameters will be either "species=userSelection" or "near=123,123&radius=25" to build the whaleAPI URL based on user selection
+function initMap(parameters) {
         $.ajax({
-          url: "http://hotline.whalemuseum.org/api.json?species=" + species + "&limit=1000",
+          url: "https://hotline.whalemuseum.org/api.json?" + parameters + "&limit=1000",
           method: "GET",
         }).then(function(response) {
             totalWhales = response;
             
         }).then(function() {
+            console.log("initMap is running! and parameters var is: " + parameters);
             initialMap = new google.maps.Map(document.getElementById('map'), {
-                zoom: 2,
-                center: new google.maps.LatLng(2.8,-187.3),
+                zoom: 6,
+                center: new google.maps.LatLng(48.5159,-123.1524),
                 mapTypeId: 'terrain'
```

Comment:
Since all the results from the whales API are centered around Seattle, I changed the initial map to be zoomed there.